scripts/generate_landings_risk_report.py

```diff
@@ -1786,12 +1786,16 @@ def calculate_maintenance_effectiveness(period):
                     2,
                 )
 
+            def format_maintenance_effectiveness(period):
```

Comment:
NIT: This also calculates besides formatting, probably the name should reflect that?